src/background/modules/OmniboxSearch.js

```diff
@@ -171,7 +171,7 @@ export async function triggerOmnixboxSearch(text, disposition) {
                 copyToClipboard: true
             });
         } else if (emojiSearch.action === "emojipedia") {
-            const resultUrl = `https://emojipedia.org/search/?q=${emojiText}`;
+            const resultUrl = `https://emojipedia.org/search/?q=${encodeURIComponent(emojiText)}`;
```

Comment:
A good catch! That is a bug fix on it's own I guess…

Reply:
Yes, one could also use the newer [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) interface instead to automatically encode this parameter.

Reply:
That is a very good idea actually! Let's use this…

Reply:
OK, will do...